tests/bindings-framework-test/bindings-framework-test.csproj

```
@@ -80,9 +67,23 @@
   </ItemGroup>
   <ItemGroup>
     <TestLibrariesInput Include="..\..\tests\test-libraries\libframework.m" />
-    <TestLibrariesOutput Include="..\..\tests\test-libraries\.libs\ios-fat\XTest.framework" />
-    <TestLibrariesOutput Include="..\..\tests\test-libraries\.libs\ios-fat\XSharedObjectTest.framework" />
-    <TestLibrariesOutput Include="..\..\tests\test-libraries\.libs\ios-fat\XSharedARTest.framework" />
+  </ItemGroup>
+  <ItemGroup>
+    <NativeReference Include="..\..\tests\test-libraries\.libs\ios-fat\XStaticArTest.framework">
+      <Kind>Framework</Kind>
+      <SmartLink>False</SmartLink>
+      <Frameworks>CoreLocation ModelIO</Frameworks>
+    </NativeReference>
+    <NativeReference Include="..\..\tests\test-libraries\.libs\ios-fat\XStaticObjectTest.framework">
+      <Kind>Framework</Kind>
+      <SmartLink>False</SmartLink>
+      <Frameworks>CoreLocation ModelIO</Frameworks>
+    </NativeReference>
+    <NativeReference Include="..\..\tests\test-libraries\.libs\ios-fat\XTest.framework">
+      <Kind>Framework</Kind>
+      <SmartLink>False</SmartLink>
+      <Frameworks>CoreLocation ModelIO</Frameworks>
```

I think this should work, that would avoid the xharness changes:

```suggestion
      <Frameworks Condition="'$(TargetFrameworkIdentifier)' != 'Xamarin.WatchOS'">CoreLocation ModelIO</Frameworks>
      <Frameworks Condition="'$(TargetFrameworkIdentifier)' == 'Xamarin.WatchOS'">CoreLocation</Frameworks>
```